tests/integration/test_factor_service_cadence.py

```
@@ -0,0 +1,253 @@
+    depth_orderbook: dict[str, list[dict[str, float]]]
+    depth_orderbook = {
+        "bids": [{"price": 0.39, "size": 100.0}],
+        "asks": [{"price": 0.41, "size": 50.0}],
+    }
+    empty_orderbook: dict[str, list[dict[str, float]]]
+    empty_orderbook = {"bids": [], "asks": []}
+
+    await _seed_market(
+        store,
+        market_id="factor-depth",
+        token_id="factor-depth-token",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "token_id"

(S106)